packages/strapi/lib/Strapi.js

```diff
@@ -42,6 +42,8 @@ class Strapi {
     this.app = new Koa();
     this.router = new Router();
 
+    this.server = http.createServer(this.handleRequest.bind(this));
+
```

Comment:
Would `this.server = http.createServer(this.app.callback());` have worked?

Reply:
Nope :)